src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +32,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /** The feedback message to be shown to the user. Contains a description of the execution result */
+    public String getFeedbackToUser() {
```

Comment:
You have added a getter method.
Don't forget to update the place where the `.feedbackToUser` is used to the getter method.